tests/math/sequence/test_sequential_funcs.py

```diff
@@ -8,7 +8,7 @@
 from pytest import raises
 
 import mygrad
-from mygrad import amax, amin, cumprod, cumsum, mean, prod, std, sum, var
+from mygrad import amax, amin, cumprod, cumsum, mean, _canonicalize_weights, average, prod, std, sum, var
```

Comment:
Move the import of `_canonicalize_weights` to be directly from `mygrad.math.sequential.funcs`